modin/core/dataframe/algebra/default2pandas/groupby.py

```diff
@@ -149,23 +149,19 @@ def build_aggregate_method(cls, key):
         def fn(
             df,
             by,
-            groupby_args,
+            axis,
+            groupby_kwargs,
             agg_args,
-            axis=0,
-            is_multi_by=None,
+            agg_kwargs,
             drop=False,
-            **kwargs
+            **kwargs,
         ):
             """Group DataFrame and apply aggregation function to each group."""
             by = cls.validate_by(by)
 
-            grp = df.groupby(by, axis=axis, **groupby_args)
+            grp = df.groupby(by, axis=axis, **groupby_kwargs)
             agg_func = cls.get_func(grp, key, **kwargs)
-            result = (
-                grp.agg(agg_func, **agg_args)
-                if isinstance(agg_func, dict)
-                else agg_func(grp, **agg_args)
```

Comment:
there are now no branches that return a dictionary in the `cls.get_func`, so removed this if-statement